static/_redirects

```diff
@@ -1,10 +1,13 @@
 # Redirect rules for Netlify
 # Although splats are supported, e.g. /* /new/path/:splat, regex isn't yet
 
-# Serve schema files located in the cnabio/cnab-spec repository
+# Serve latest schema files located in the cnabio/cnab-spec repository
 /v1/bundle.schema.json https://raw.githubusercontent.com/cnabio/cnab-spec/master/schema/bundle.schema.json
 /v1/claim.schema.json https://raw.githubusercontent.com/cnabio/cnab-spec/master/schema/claim.schema.json
 /v1/definitions.schema.json https://raw.githubusercontent.com/cnabio/cnab-spec/master/schema/definitions.schema.json
 /v1/dependencies.schema.json https://raw.githubusercontent.com/cnabio/cnab-spec/master/schema/dependencies.schema.json
 /v1/relocation-mapping.schema.json https://raw.githubusercontent.com/cnabio/cnab-spec/master/schema/relocation-mapping.schema.json
-/v1/status.schema.json https://raw.githubusercontent.com/cnabio/cnab-spec/master/schema/status.schema.json
\ No newline at end of file
+/v1/status.schema.json https://raw.githubusercontent.com/cnabio/cnab-spec/master/schema/status.schema.json
+
+# Serve versioned schema files from the CDN source
+/schema/* https://cdn.cnab.io/schema/*
```

Comment:
should this be 
```suggestion
/schema/* https://cdn.cnab.io/schema/:splat
```

https://docs.netlify.com/routing/redirects/redirect-options/#splats

Reply:
Yes! 🙏 Thank you!